Rubrik/Public/New-RubrikDatabaseMount.ps1

```diff
@@ -0,0 +1,106 @@
+#Requires -Version 3
+function New-RubrikDatabaseMount
+{
+  <#  
+      .SYNOPSIS
+      Create a new Live Mount from a protected database
+      
+      .DESCRIPTION
+      The New-RubrikDatabaseMount cmdlet is used to create a Live Mount (clone) of a protected database and run it in an existing database environment.
+      
+      .NOTES
+      Written by Mike Fal for community usage from New-RubrikMount
+      Twitter: @Mike_Fal
+      GitHub: MikeFal
+      
+      .LINK
+      https://github.com/rubrikinc/PowerShell-Module
+
+      .EXAMPLE
+      New-RubrikDatabaseMount -id $db.id -targetInstanceId $db.instanceId -mountedDatabaseName 'BAR-LM' -recoveryDateTime (Get-date (Get-RubrikDatabase -id $db.id).latestRecoveryPoint)
+      Creates a new database mount named BAR on the same instance as the source database, using the most recent recovery time for the database. 
+      
+      $db=Get-RubrikDatabase -HostName FOO -Instance MSSQLSERVER -Database BAR
+
+  #>
+
+  [CmdletBinding(SupportsShouldProcess = $true,ConfirmImpact = 'High')]
+  Param(
+    # Rubrik id of the database
+    [Parameter(Mandatory = $true,ValueFromPipelineByPropertyName = $true)]
+    [String]$id,
+    # ID of Instance to use for the mount
+    [Alias('InstanceId')]
+    [String]$targetInstanceId,
+    # Name of the mounted database
+    [Alias('DatabaseName')]
+    [String]$mountedDatabaseName,
+    # Recovery Point desired in the form of Epoch with Milliseconds
+    [Parameter(ParameterSetName='Recovery_timestamp')]
+    [int64]$timestampMs,
+    # Recovery Point desired in the form of DateTime value
+    [Parameter(ParameterSetName='Recovery_DateTime')]
+    [datetime]$recoveryDateTime,
+    # Rubrik server IP or FQDN
+    [String]$Server = $global:RubrikConnection.server,
+    # API version
+    [String]$api = $global:RubrikConnection.api
+  )
+
+  Begin {
+
+    # The Begin section is used to perform one-time loads of data necessary to carry out the function's purpose
+    # If a command needs to be run with each iteration or pipeline input, place it in the Process section
+    
+    # Check to ensure that a session to the Rubrik cluster exists and load the needed header data for authentication
+    Test-RubrikConnection
+    
+    #Set epoch for recovery time calculations
```

Comment:
If you roll this into a private function (earlier comment) it allows for code re-use :)